packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,567 @@
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Dont>
+</DoDont>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/grid
+```
+
+#### Usage
+
+```jsx
+import {Grid, Column} from '@twilio-paste/grid';
+
+<Grid>
+  <Column>
+    Foo
+  </Column>
+</Grid>
+```
+
+#### Grid Props
+
+| Prop      | Type                                         | Description                                                                                                                         | Default |
+| ----------| ---------------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| gutter?   | `ResponsiveValue<keyof ThemeShape['space']>` | Sets the spacing on both sides of a Column. Use a spacing token that represents half of the total space you need between 2 Columns. |         |
+| vertical? | `boolean, ResponsiveValue<boolean>`          | Stacks the Columns                                                                                                                  |         |
+
+#### Column Props
+
+| Prop      | Type                                         | Description                                                                                                                         | Default |
+| ----------| ---------------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| span?     | `number, ResponsiveValue<number>`            | Sets the width of Columns (how many columns a column can span)                                                                      |         |
+| offset?   | `number, ResponsiveValue<number>`            | Sets the left margin of a Column (how many columns a column is offset)                                                              |         |
+
+### Sketch
+
+Add Grid symbols from the “Paste components” project to your own artboards to help you align content.
+
+Additionally, each Paste container component (*e.g.*, Card, Modal) symbol in Sketch has an invisible grid symbol named “📐 Grid” that you can replace with visible grid symbols.
+
+We use grid symbols instead of Sketch Layout Settings for two reasons:
+
+1. Resizing a symbol better represents the behavior of a fluid grid in a responsive layout. You might be designing for multiple screen widths in a single Sketch file. Sketch Layout Settings, however, restricts you to a single setting of grid width, column width, gutter width, and number of columns for each file.
+2. Symbols allow you to place a grid within a component or container on an artboard, whereas Sketch Layout Settings can be applied to a whole artboard only.
```

Comment:
"can be applied to a whole artboard only" --> "can be applied only to a whole artboard"